salt/_modules/kubeconfig.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+
+import yaml
+
+__virtualname__ = 'metalk8s_kubeconfigs'
```

Comment:
Looks like this is also done in the related state, but: let's try to keep Python module names and `__virtualname__` the same.